src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -248,8 +248,39 @@ import Wrapper from '../components/Wrapper.astro';
 </Wrapper>
 ```
 
-
+:::tip
 Use a `slot="my-slot"` attribute on the child element that you want to pass through to a matching `<slot name="my-slot" />` placeholder in your component.
+:::
+
+To pass multiple HTML elements into a component's `<slot/>` placeholder without a wrapping `<div>`, use the `slot=""` attribute on [Astro's `<Fragment/>` component](/en/reference/api-reference/#fragment-):
+
+```astro title="src/components/CustomTable.astro" "<slot name="header"/>" "<slot name="body"/>"
+---
+// Create a custom table with named slot placeholders for head and body content
+---
+<table class="bg-white">
+  <thead class="sticky top-0 bg-white"><slot name="header"/></thead>
+  <tbody class="[&_tr:nth-child(odd)]:bg-gray-100"><slot name="body"/></tbody>
+</table>
+```
+
+Inject multiple rows and columns of HTML content using a `slot=""` attribute to specify the `"header"` and `"body"` content. Individual HTML elements can also be styled:
+
+```astro title="src/components/StockTable.astro" {5-7, 9-13} '<Fragment slot="header">' '<Fragment slot="body">'
+---
+import CustomTable from './CustomTable.astro';
+---
+<CustomTable>
+  <Fragment slot="header"> <!-- pass table header -->
+    <tr><th>Product name</th><th>Stock units</th></tr>
+  </Fragment>
+
+  <Fragment slot="body"> <!-- pass table body -->
+    <tr><td>Flip-flops</td><td>64</td></tr>
+    <tr><td>Boots</td><td>32</td></tr>
+    <tr><td>Sneakers</td><td class="text-red-500">0</td></tr>
+  </Fragment>
+</CustomTable>
```

Comment:
``````suggestion
</CustomTable>
```
``````

Reply:
Good eye! I was just looking at that in the preview!